scripts/encrypt/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "keywords": [],
   "dependencies": {
-    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#296592c",
+    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#722e1c7",
```

Comment:
Seems sensible to bump this at the same time!

Reply:
Honestly might not be a bad idea to regenerate lockfiles in planx-core too at some point!